Src/ReleaseHistory.md

```diff
@@ -38,6 +38,8 @@
   removing invalid characters (`,`, `=`, `|`, `&`, `[`, `]`, `>`) from `Id` and
   `Resource`.
   [#555](https://github.com/microsoft/sarif-pattern-matcher/pull/555)
+- SDK: Expanding `HttpMockHelper` to be able to mock `HttpResponseMessage`.
```

Comment:
>HttpMockHelper [](http://example.com/codeflow?start=18&length=14)

This kind of change is generally not customer impacting, why are you documenting it?

Reply:
thanks for reviewing